benchmark-service/benchmark/benchmark.py

```diff
@@ -245,6 +271,7 @@ async def on_startup(app):
     app['github_client'] = gidgethub.aiohttp.GitHubAPI(aiohttp.ClientSession(),
                                                        'hail-is/hail',
                                                        oauth_token=oauth_token)
+    app['batch_client'] = await bc.BatchClient(billing_project='test')
```

Comment:
This is fine for now, but ultimately we want this to be benchmark. Remind me before we start running the full submit function. Right now, this requires a Batch database migration.

Reply:
I'll work on a PR now that handles this.

Reply:
See #9668